packages/sales-api-service/src/schema/staging-exception.schema.js

```diff
@@ -54,6 +54,8 @@ const schemaObject = {
   }).optional()
 }
 
+const transactionDate = dateSchema.description('An ISO8601 compatible date string defining when the transaction was completed')
```

Comment:
I think this might be personal preference, but because it's a const outside of a function, I think this should be snake case in capitals, so TRANSACTION_DATE.

Reply:
Ah yes, good point!